### ISSUE TITLE
Email editor strips up some tags that are required for email to be responsive

### ISSUE THREAD

**Issue by snowcoda** [mautic/mautic#2370]
What type of report is this:

| Q | A |
| --- | --- |
| Bug report? |  |
## Description:

I'm using a template that I created and tested using MailChimp and Litmus. The template is a responsive one that adapts to screen width on most devices. Unfortunately, the editor removes some of the required HTML from the template head on save. I've tried adding it into the editor manually and then resaving, but it still gets removed.

Here's the content that gets taken out / replaced:

```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
    <head>
        <!--[if gte mso 9]><xml>
            <o:OfficeDocumentSettings>
                <o:AllowPNG/>
                <o:PixelsPerInch>96</o:PixelsPerInch>
            </o:OfficeDocumentSettings>
        </xml><![endif]-->
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```
## If a bug:

| Q | A |
| --- | --- |
| Mautic version | 2.1 |
| PHP version | 7.08 |
### Steps to reproduce:
1. Template added as a theme
2. Create new email and view HTML
3. Markup from header as detailed above is not present / has been replaced.
4. Manually paste missing HTML back in.
5. Click Save.
6. HTML is again removed / replaced.
### Log errors:

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by escopecz** [comment]
It seems that this issue was fixed in https://github.com/froala/wysiwyg-editor/issues/1455 a day ago. We have to wait for Froala to release a new version. 

Is the issue only the comment or the attributes in the HTML tag as well? If so, please, create the issue at https://github.com/froala/wysiwyg-editor/issues

**Comment by snowcoda** [comment]
Good to hear that it sounds like Froala is working on the issue.

The issue is the comment and the attributes in the HTML tag. I'll create the issue with Froala.

Many thanks

Paul

**Comment by escopecz** [comment]
Fixed in 2.3
